perceval/components/processor.py

```diff
@@ -54,6 +54,7 @@ class Processor(AProcessor):
         >>> p = Processor("SLOS", BS() // PS() // BS())
 
     :param source: the Source used by the processor (defaults to perfect source)
+    :param name: a textual name for the processor (defaults to "Local processor")
     """
```

Comment:
Do we want to enrich the documentation and be standard to the sphix documentation ?

    """
    Generic definition of processor as a source + components (both unitary and non-unitary) + ports
    + optional post-processing logic

    :param backend: Name or instance of a simulation backend
    :type backend (Union[ABackend, str]):
    :param m_circuit: can either be:

        - an int: number of modes of interest (MOI). A mode of interest is any non-heralded mode.
        >>> p = Processor("SLOS", 5)

        - a circuit: the input circuit to start with. Other components can still be added afterwards with `add()`
        >>> p = Processor("SLOS", BS() // PS() // BS())

        , defaults to None.

    :type m_circuit (Union[int, ACircuit], optional):


    :param source: the Source used by the processor, defaults to perfect source.
    :type source (Source, optional):
    :param name: a textual name for the processor, defaults to "Local processor".
    :type name (str, optional):
    """
    
![image](https://github.com/Quandela/Perceval/assets/137297230/79999e1f-bca2-4d5d-aec9-dffc8cc29fcb)